pkg/apis/messaging/v1/subscription_validation_test.go

```diff
@@ -266,6 +267,22 @@ func TestSubscriptionSpecValidation(t *testing.T) {
 			Delivery:   getDelivery(backoffDelayInvalid),
 		},
 		want: apis.ErrInvalidValue(backoffDelayInvalid, "delivery.backoffDelay"),
+	}, {
+		name: "non-empty Channel namespace",
+		c: &SubscriptionSpec{
+			Channel: duckv1.KReference{
+				Kind:       channelKind,
+				APIVersion: channelAPIVersion,
+				Name:       channelName,
+				Namespace:  channelNamespace,
+			},
+			Subscriber: getValidDestination(),
+		},
+		want: func() *apis.FieldError {
+			fe := apis.ErrDisallowedFields("channel.namespace")
```

Comment:
I found that the unit test failed if I wrote `fe := apis.ErrDisallowedFields("namespace")` but I wrote this line in subscription_channellable_validation.go. I'm just curious why is `channel.namespace` needed here?

Reply:
I think it is because the field is technically at the path `.channel.namespace`, it's not just at the path `.namespace`